test/data/navigator/ua-remote-iframe.html

```diff
@@ -5,15 +5,6 @@
     <title></title>
   </head>
   <body>
-    <script>
-      window.onmessage = function(event) {
-        if (navigator.userAgent == event.data) {
-          document.title = "pass";
-        }
-      }
-      var iframe = document.createElement("iframe");
-      iframe.src = document.location.href.replace("b.com", "z.com").replace("remote-", "");
-      document.body.appendChild(iframe);
-    </script>
+    <script src="ua-remote-iframe.js"></script>
```

Comment:
this extraction to `.js` is required, otherwise we get CSP errors when launching this document as an offscreen one.